### PR TITLE
Version number

### PR DESCRIPTION
**Pull request by mhyder1**
# Proposed Changes

Describe the big picture of your changes here.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

- [ ] My changes generate no new warnings
- [ ] Logging has been added where applicable (Error handling, user-driven events, API requests/responses)

- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] Tests meet the test coverage threshold

## Reviewer Checklist

- [ ] Is the code easy to understand?
- [ ] Are the changes well documented?

- [ ] Can you think of any use case in which the code does not behave as intended?
- [ ] Can you think of any inputs or external events that could break the code?

- [ ] Should any logging or debugging information be added or removed?

## Further Comments

If this is a relatively large or complex change, kick off the discussion by explaining how you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/app/src/components/homepage/index.tsx [hunk 1]
```diff
@@ -19,6 +20,8 @@ const Wrapper = ({ children }: PropsWithChildren<{}>) => (
 );
 
 export const HomePage = () => {
+  // const { version } = versionNumber
+  // console.log({version})
```
Comment (KaemonIsland) [inline packages/app/src/components/homepage/index.tsx:24]:
```suggestion
```

Should these lines be removed?

File: packages/app/src/components/homepage/index.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { Grid, Button } from '@material-ui/core';
 import React, { PropsWithChildren } from 'react';
 import { Link } from 'react-router-dom';
 import { Content, Header, InfoCard, Page } from '@backstage/core-components';
+import versionNumber from '../../version.json'
```
Comment (KaemonIsland) [inline packages/app/src/components/homepage/index.tsx:5]:
```suggestion
```
Should this be removed? I'm not seeing it used anywhere

**Review by KaemonIsland — APPROVED** [review]

**Review by keyluck — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

File: packages/app/src/components/homepage/index.tsx (hunk 2 above)
Comment (mhyder1) [reply packages/app/src/components/homepage/index.tsx:5]: Correct. I need to do a little more clean up.

File: packages/app/src/components/homepage/index.tsx (hunk 1 above)
Comment (mhyder1) [reply packages/app/src/components/homepage/index.tsx:24]: definitely. updated.

**Pull request merged**